### PR TITLE
Rachel login stuffs

### PR DESCRIPTION
**Pull request by rachelhongca**
login!!

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -69,12 +71,20 @@ class ConfigClass(object):
...
-from flask_user import SQLAlchemyAdapter, UserManager
 
-db_adapter = SQLAlchemyAdapter(db, User)        # Register the User model
-user_manager = UserManager(db_adapter, app, register_form = InviteForm, 
-    register_view_function = invite)     # Initialize Flask-User
+# Create auth blueprint.
+from .auth import auth as auth_blueprint
+app.register_blueprint(auth_blueprint)
+
+#db_adapter = SQLAlchemyAdapter(db, User)        # Register the User model
```
Comment (thomaslee1911) [inline app/__init__.py]: delete commented-out code?

File: app/auth/forms.py [hunk 2]
```diff
@@ -1,7 +1,14 @@
 from app import app
+from wtforms import StringField, PasswordField, BooleanField, SubmitField
+from wtforms.validators import Required, Email
 from magic_numbers import *
 from flask import current_app
 from flask.ext.wtf import Form
+<<<<<<< HEAD
```
Comment (weronica) [inline app/auth/forms.py]: Remove merge conflict marks.

File: app/auth/forms.py [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
+from wtforms.validators import InputRequired, Length, Email, Optional, EqualTo
+from wtforms import TextField
+from ..models import User
+=======
```
Comment (weronica) [inline app/auth/forms.py]: Remove merge conflict marks.

File: app/auth/views.py [hunk 4]
```diff
@@ -3,15 +3,42 @@
 from flask import render_template, flash, redirect, session, url_for, request, \
 	g, jsonify, current_app, Response
 from . import auth
+<<<<<<< HEAD
```
Comment (weronica) [inline app/auth/views.py]: Remove merge conflict marks, and make sure that the site still runs.

File: app/auth/views.py [hunk 5]
```diff
@@ -3,15 +3,42 @@
...
+from ..models import InvitedUser, User
+from flask_login import current_user, login_user
+from flask.ext.login import login_required, logout_user
+from forms import InviteAdminForm, LoginForm, ChangePasswordForm
+=======
```
Comment (weronica) [inline app/auth/views.py]: Remove merge conflict marks, and make sure that the site still runs.

**Pull request merged**